BraveRewardsUI/Publisher/PublisherSummaryView.swift

```diff
@@ -31,6 +33,8 @@ class PublisherSummaryView: UIView {
     $0.setTitle(Strings.PublisherSendTip.uppercased(), for: .normal)
   }
   
+  var pushViewControllerHandler: ((UIViewController) -> Void)?
```

Comment:
Once `DetailActionableRow` becomes a `Button`, you can remove this and just `addTarget` directly on it

Reply:
Done!